src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age
+    && <p className="Person__age">{`I am ${person.age}`}</p>
+    }
+
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my `}
+        {person.sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    )
+      : (
+        <p className="Person__partner">
+          I am not married
+        </p>
+      )}
```

Comment:
```suggestion
    {person.isMarried ? (
      <p className="Person__partner">
        {`${person.partnerName} is my `}
        {person.sex === 'm' ? 'wife' : 'husband'}
      </p>
    )
      : (
        <p className="Person__partner">
          I am not married
        </p>
      )}
```
Consider moving this logic into a separate function from jsx